### DISCUSSION TITLE
Any plan on a build for Apple Silicon?

### DISCUSSION THREAD

**Discussion by jdp1024** [MiKTeX/miktex#1158] [General]
I compiled all the binary tools (those in sandbox/bin) for darwin-arm64 (Apple Silicon) and replaced them over the ones included in the official package. Big performance boost was received: for a 45000-line tex file which turns into a PDF file with 1160 pages and over 13900 chessboards, the compilation time dropped from 11m40s to 7m18s.

I failed to build the console app with Qt 5.15.5 (the homebrew installation): too many compilation errors about the template usage **inside** Qt itself. My knowledge to Qt is very limited so I am unable to fix them.

Would someone could contribute a recipe to build MiKTeX against the latest homebrew Qt? Thank in advance.

**Comment by qns7** [comment]
I am very interested in this also. Can someone "official" please say something about this?

**Comment by bobpaw** [comment]
I would also like to voice my support for an Apple Silicon build. Support will soon end for Rosetta and MikTeX will only be usable if built from source.

**Comment by wikror** [comment]
Hi, I wanted to also ask about plans for support for Apple Silicon! This is by far the best TeX distribution I have used and a build for AS architecture once the support for Intel-based apps is withdrawn would be immensely useful!
